### PR TITLE
Create README for fuzzing corpus folder 

### PR DESCRIPTION
**Pull request by Google-Autofuzz**

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -363,12 +362,13 @@ cc_test(
...
 cc_binary(
-    testonly = 1,
     name = "file_descriptor_parsenew_fuzzer",
+    testonly = 1,
     srcs = ["tests/file_descriptor_parsenew_fuzzer.cc"],
     copts = CPPOPTS + select({
         "//conditions:default": [],
-        ":fuzz": ["-fsanitizer=fuzzer,address"],
+        ":fuzz": [],
+        #       ":fuzz": ["-fsanitizer=fuzzer,address"],
```
Comment (haberman) [inline BUILD]: I don't think you want this to be empty?

Comment (Google-Autofuzz) [reply BUILD]: You are right that should not be empty. I am trying to fix the last error message. The only error I am getting now in Travis CI when PR to Google/OSS-Fuzz was during the libfuzzer undefined test. It looks like the fuzz target wasn't built in "--sanitized undefined" config. So far everything works for libfuzzer address and memory. 

https://travis-ci.org/google/oss-fuzz/jobs/550517792#L2272

File: tests/corpus/temp.cc [hunk 2]
```diff
@@ -0,0 +1 @@
+// Hello World
```
Comment (haberman) [inline tests/corpus/temp.cc]: What is the purpose of this .cc file? Can we remove it?

I assume the corpus directory is for storing the fuzz corpus? Can we rename it `tests/fuzz_corpus`?

Comment (Google-Autofuzz) [reply tests/corpus/temp.cc]: Yes, you can remove it and rename it to `tests/fuzz_corpus`.

**Pull request merged**